app/experimenter/visualization/api/v1/views.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from django.core.files.storage import default_storage
+from django.http import HttpResponse
+from django.views import View
+
+from experimenter.experiments.models import Experiment
+
+
+class VisualizationView(View):
+    def load_data_from_gcs(self, filename):
+        return (
+            json.loads(default_storage.open(filename).read().decode("utf8"))
```

Comment:
Since this is doing a decode, in theory it could fail, maybe add a test case for data that isn't valid utf8 and make sure it gets caught nicely?

Reply:
I'm not sure why I had the decode there, but I actually don't need it so I removed it.